static/app/components/devtoolbar/components/indicatorBadge.tsx

```diff
@@ -25,12 +23,12 @@ const counterCss = css`
   border: 1px solid transparent;
   box-sizing: content-box;
   color: var(--gray100);
-  height: 1rem;
+  height: 0.55rem;
   line-height: 1rem;
   position: absolute;
-  right: -6px;
-  top: -6px;
-  width: 1rem;
+  right: 2px;
```

Comment:
Css learning question: Is right and top used to position it relative to the panel icon? How does position:absolute work here?

Reply:
@aliu39 
> Is right and top used to position it relative to the panel icon? 

yes!

> How does position:absolute work here?

it works so that our `top` and `left` are relative to the icon itself, since we want the dot on top of the icon.


![image](https://github.com/user-attachments/assets/1230cc54-a98d-47a5-81f4-74d7698f2c98)

relative:

<img width="48" alt="SCR-20240815-izsg" src="https://github.com/user-attachments/assets/6ece1b09-7554-4459-be4c-21c26569a795">

absolute:


<img width="42" alt="SCR-20240815-izoq" src="https://github.com/user-attachments/assets/4951534a-f88b-4da9-92a0-4ca0858783c5">

Reply:
🔥 🔥 thanks for the diagrams!